met-web/src/components/engagement/admin/create/authoring/AuthoringFeedback.tsx

```diff
@@ -0,0 +1,281 @@
+    };
+    const formItemContainerStyles = {
+        padding: '2rem 1.4rem !important',
+        margin: '1rem 0',
+        borderRadius: '16px',
+    };
+    const metLabelStyles = {
+        fontSize: '0.95rem',
+    };
+    const buttonStyles = {
+        height: '2.6rem',
+        borderRadius: '8px',
+        border: 'none',
+        padding: '0 1rem',
+        minWidth: '8.125rem',
+        fontSize: '0.9rem',
+    };
+    const widgetPreviewStyles = {
+        margin: '2rem 4rem 4rem',
+        display: 'flex',
+        minHeight: '18rem',
+        border: '2px dashed rgb(122, 120, 118)',
+        alignItems: 'center',
+        justifyContent: 'center',
+        borderRadius: '16px',
+    };
+    const conditionalSelectStyles = {
+        width: '100%',
+        backgroundColor: colors.surface.white,
+        borderRadius: '8px',
+        boxShadow: '0 0 0 1px #7A7876 inset',
+        lineHeight: '1.4375em',
+        height: '48px',
+        marginTop: '8px',
+        padding: '0',
+    };
+    const toolbar = {
+        options: ['inline', 'list', 'link', 'blockType', 'history'],
+        inline: {
+            options: ['bold', 'italic', 'underline', 'superscript', 'subscript'],
+        },
+        blockType: { options: ['Normal', 'H2', 'H3', 'Blockquote'] },
+        list: { options: ['unordered', 'ordered'] },
+    };
+
+    const handleEditorChange = (newEditorState: EditorState) => {
+        const plainText = newEditorState.getCurrentContent().getPlainText();
+        setEditorState(newEditorState);
+        setBodyText(plainText);
+    };
+
+    const handleWidgetChange = (event: SelectChangeEvent<string>) => {
+        setCurrentWidget(event.target.value);
+    };
+
+    const handleRemoveWidget = () => {
+        if ('' === currentWidget) {
+            return;
+        } else {
+            setCurrentWidget('');
+        }
+    };
+
+    return (
+        <Grid container sx={{ maxWidth: '700px', mt: '1rem' }} direction="column">
+            <Grid item sx={{ mt: '1rem' }}>
+                <MetHeader3 style={metHeader3Styles}>Primary Content (Required)</MetHeader3>
+                <EyebrowText style={eyebrowTextStyles}>
+                    This section of content should provide a brief overview of what your engagement is about and what
+                    you would like your audience to do.
+                </EyebrowText>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
+                <label htmlFor="section_heading">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Section Heading <span style={{ fontWeight: 'normal' }}>(Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
+                        Your section heading should be descriptive, short and succinct.
+                    </EyebrowText>
+                    <TextField
```

Comment:
Nat once mentioned that our TextField component attempts to auto generate a label. Could you let me know what the label for this text field will be ? Also, will screen reader users understand this field is required too?

If there's no label here then I'd recommend we make the h3 above into one and just style it to match the designs

Reply:
I just looked and I'm not seeing an auto-generated label for the TextField component in the browser. Perhaps I need to use a specific prop. If it's ok with you, I think this is something that could be addressed in the individual authoring page tickets.

Reply:
I appreciate that you've already done a lot of work for the authoring flow but I think it'd be best to handle these now. We already have to update the other tickets to describe what's been done in this one and I'm worried about adding another complicating factor like "rework the HTML to better serve screen readers." If we're publishing the HTML then we should do it right from the get-go.

That said, I don't think it'd be too much work to change each of those elements to a label, updating the styling if needed, then associate them with the input fields. It should be the same fix for each of the pages. This would be one approach.

What's important is just that screen readers read out all this information when a user tabs onto a field - however you want to approach that

Reply:
I made some other changes that will definitely help the way screen readers read the page. I've now tested it with Pericles and the mac OS voice service. The h3s that were form labels have been converted to form labels. I also removed the alt tag on the image in the "Preview" button and added an aria-invisible tag to it.